gen-windows-VS2019.bat

```diff
@@ -0,0 +1,12 @@
+@echo off
+
+REM Generate project files
+mkdir build-windows-VS2019
+pushd build-windows-VS2019
+
+IF "%CI%"=="" (
+	cmake -G "Visual Studio 16 2019" ..
```

Comment:
Why is there no `Win64`

Reply:
is normal for 2019, is removed on cmake, it compile default on 64 bit now